src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -122,9 +122,6 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
                     name: "PSNativePSPathResolution",
                     description: "Convert PSPath to filesystem path, if possible, for native commands"),
-                new ExperimentalFeature(
-                    name: "PSNotApplyErrorActionToStderr",
```

Comment:
I found "PSNotApplyErrorActionToStderr" in NativeCommandProcessor.Tests.ps1

Could you confirm that you search and remove all feature names from test files too?

Reply:
I fixed that in another commit already, I don't see any instances in tests